integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse;
+
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * This class holds all fuse-related options used by libfuse and AlluxioFuse.
+ */
+public final class AlluxioFuseFileSystemOpts {
+  private final String mAlluxioPath;
+  private final String mFsName;
+  private final Class mFuseAuthPolicyClass;
+  private final Optional<String> mFuseAuthPolicyCustomGroup;
+  private final Optional<String> mFuseAuthPolicyCustomUser;
+  private final int mFuseMaxPathCached;
+  private final int mFuseUmountTimeout;
+  private final boolean mIsDebug;
+  private final List<String> mLibfuseOptions;
+  private final boolean mMetadataCacheEnabled;
+  private final String mMountPoint;
+  private final boolean mSpecialCommandEnabled;
+  private final long mStatCacheTimeout;
+  private final boolean mUserGroupTranslationEnabled;
+
+  /**
+   * Constructs an {@link AlluxioFuseFileSystemOpts} with only Alluxio cluster configuration.
+   *
+   * @param conf     Alluxio cluster configuration
+   * @return AlluxioFuseFileSystemOpts
+   */
+  public static AlluxioFuseFileSystemOpts create(AlluxioConfiguration conf) {
```

Comment:
Usually if there are multiple create functions, only one do the real thing, the others just add/remove/modify parameters
some of the logics in these two create() functions are the same, try to use the `Create( AlluxioConfiguration conf, AlluxioFuseCliOpts fuseCliOpts) ` function directly. e.g. 

public static AlluxioFuseFileSystemOpts create(AlluxioConfiguration conf) {
    return create(conf, <am empty version of AlluxioFuseCliOpts>);
}